graphql_client/Cargo.toml

```diff
@@ -56,6 +50,8 @@ reqwest = "^0.9"
 wasm-bindgen-test = "^0.2"
 
 [features]
+default = ["graphql_query_derive"]
+noderive = []
```

Comment:
`graphql_client = { default-features = false, features = ["noderive"]` would be the same as  `graphql_client = { default-features = false`, is there any particular reason to include this?

Reply:
No reason - I've dropped it now that you point it out.